crates/pipeline-types/src/transport/delta_table.rs

```diff
@@ -3,11 +3,32 @@ use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use utoipa::ToSchema;
 
+/// Delta table write mode.
+///
+/// Determines how the Delta table connector handles an existing table at the target location.
+#[derive(Default, Debug, Clone, Eq, PartialEq, Deserialize, Serialize, ToSchema)]
+pub enum DeltaTableWriteMode {
+    /// New updates will be appended to the existing table at the target location.
+    #[default]
```

Comment:
Is this also the interaction that was before this PR?

Reply:
yes, this was the only supported behavior.